CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 The noteworthy changes for each ExMachina version are included here. For a
 complete changelog, see the git history for each version via the version links.
 
+**To see the dates a version was published after 0.6.0**, see the
```

Comment:
It was annoying to have to add the date. Hex already does it so I figure we could just link to that page instead.